common/minicurl.hh

```diff
@@ -39,7 +39,8 @@ public:
   MiniCurl& operator=(const MiniCurl&&) = delete;
   void setURLData(const std::string& url, const MiniCurlHeaders& headers);
   std::string getURL(const std::string& url, const MiniCurlHeaders& headers);
-  void setCurlOption(int option, ...);
+  void setCurlOptionLong(int option, long l);
```

Comment:
I had to get rid of the variadic version because there was no varargs version of the underlying curl function to call. I really should just replace this with a template I guess.

Reply:
Replaced with a templatized version